zpl/label.py

```diff
@@ -180,15 +186,24 @@ def write_field_number(self, number, name=None, char_height=None, char_width=Non
                 "characters and spaces"
             self.code += '"%s"' % name
 
+    def barcode_field_default(self, module_width, bar_width_ratio, height):
+        self.code += '^BY%s,%s,%s' % (module_width * self.dpmm,
+                                      bar_width_ratio,
+                                      height * self.dpmm)
+
+    def field_orientation(self, orientation):
```

Comment:
To optionally support the alignment parameter would be useful.

Reply:
I've added the justification parameter with a default value of None for origin, labelhome, and field_orientation.  Following suit with the rest of the code, I haven't set up any enums or translation from the ZPL parameter codes, so 0=left justify, 1=right justify, 2=auto justify, and leaving it out has its own specific behavior from the spec.  This might be confusing with the other justification parameters from the spec that have the codes L, C, R, and J.  I don't see a consistent way to get around the issue without setting up translation, and since the module seems to assume that the implementor has a knowledge of ZPL parameter codes anyway, I'm inclined to leave it like this without enums or translation, but I want to get your opinion on the matter.  Thanks!

Reply:
Courtesy link to ZPL spec I'm looking at: https://support.zebra.com/cpws/docs/zpl/13979l-010_ra.pdf

Reply:
> I've added the justification parameter with a default value of None for origin, labelhome, and field_orientation.
yes, that's what I was going for. :)

> [...] I don't see a consistent way to get around the issue without setting up translation
I wanted to keep the module as simple as possible, but it is annoying that ZPL uses different codes for the same behaviour. Another way to approach this could be describing the parameters in the docstring of each function.